docs/ops/upgrading.md

```diff
@@ -172,7 +172,7 @@ First major step in job migration is taking a savepoint of your job running in t
 You can do this with the command:
 
 {% highlight shell %}
-$ bin/flink savepoint :jobId [:targetDirectory]
+$ ./bin/flink savepoint <jobId> [savepointDirectory]
```

Comment:
`savepointDirectory` has specific semantics in the documentation and is assumed to be the directory with which one can start a job from again.
The directory passed here _does not fit these semantics_ as a `savepointDirectory` is created *within* the `targetDirectory`.

Please revert.